README.md

```diff
@@ -28,16 +83,15 @@ R2D2 takes input in the form of the hash of Android Pay token values:
 }
 ```
 
-and the merchant's private key private key (which is managed by a third-party such as a gateway or independent processor like [Spreedly](https://spreedly.com)).
+and the merchant's private key (which is managed by a third-party such as a gateway or independent processor like [Spreedly](https://spreedly.com)).
 
 ```ruby
-require "android_pay"
+require 'R2D2'
```

Comment:
Why did we capitalize this!? Too late now I guess 💁🏻‍♂️

Reply:
not too late!! _fixed_